CONTRIBUTING.rst

```diff
@@ -109,5 +109,5 @@ Before you submit a pull request, check that it meets these guidelines:
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring.
 3. The pull request should work for Python 2.7, 3.3, 3.4 and 3.5. Check
```

Comment:
Right. It was the case in the past but indeed due to #21 this is no longer a requirement.